python/interpret_community/widget/explanation_dashboard.py

```diff
@@ -0,0 +1,183 @@
+from gevent.pywsgi import WSGIServer
+from flask import Flask, render_template, request
+from IPython.display import display, IFrame
+import threading
+import socket
+import requests
+import os
+import json
+from .explanation_dashboard_input import ExplanationDashboardInput
+
+"""Explanation Dashboard Class.
+
+:param explanation: An object that represents an explanation.
+:type explanation: ExplanationMixin
+:param model: An object that represents a model. It is assumed that for the classification case
+    it has a method of predict_proba() returning the prediction probabilities for each
+    class and for the regression case a method of predict() returning the prediction value.
+:type model: object
+:param dataset:  A matrix of feature vector examples (# examples x # features), the same samples
+    used to build the explanation. Will overwrite any set on explanation object already
```

Comment:
(nit: slightly change the wording)
Will overwrite any set on explanation object already ->
Overwrites any existing dataset on the explanation object